packages/vuetify/src/components/VDatePicker/mixins/date-picker-table.ts

```diff
@@ -192,5 +193,13 @@ export default mixins(
         directives: [touchDirective],
       }, [transition])
     },
+    isInRange (value: string): boolean {
+      if (this.range && this.value.length === 2) {
+        const [from, to] = [...this.value as string[]].sort()
```

Comment:
Add `PropValidator<string | string[]>` to `value` prop and you should be able to remove `as string[]`

Reply:
This is super convenient.